CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.2.1] - 2017-07-19
```

Comment:
I am also kinda torn on if it should be considered a patch fix because it's really a new feature that was improperly documented. I think I am gonna consider it an enhancement rather than a bug.